### PR TITLE
Add runbooks for Nuxt response times alarms

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Related to #2504 by @sarayourfriend

## Description

Adds the run books for the general Nuxt response time alarms.

## Testing Instructions
Look for the preview in the comments and check that the instructions make sense.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: documentation/meta/monitoring/runbooks/nuxt_avg_response_time_above_threshold.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Run Book: API Production Average Response Time above threshold
```
Comment (krysal) [inline documentation/meta/monitoring/runbooks/nuxt_avg_response_time_above_threshold.md:1]: Small detail 🤏
```suggestion
# Run Book: Nuxt Production Average Response Time above threshold
```

File: documentation/meta/monitoring/runbooks/nuxt_p99_response_time_above_threshold.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Run Book: API Production Average Response Time above threshold
```
Comment (krysal) [inline documentation/meta/monitoring/runbooks/nuxt_p99_response_time_above_threshold.md:1]:
```suggestion
# Run Book: Nuxt Production Average Response Time above threshold
```

**Review by krysal — CHANGES_REQUESTED** [review]
Looks good! Just a small request here.

**Review by krysal — APPROVED** [review]
LGTM 🚀

**Review by sarayourfriend — APPROVED** [review]
LGTM!

**Comment by obulat** [comment]
I am waiting for the companion PR being approved and merged to update the alarm link here.

File: documentation/meta/monitoring/runbooks/nuxt_avg_response_time_above_threshold.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Run Book: Nuxt Production Average Response Time above threshold
+
+```{admonition} Metadata
+Status: **Unstable**
+Maintainer: @obulat
+Alarm link:
+- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/Nuxt+Production+Average+Response+Time+above+threshold?>
```
Comment (obulat) [inline documentation/meta/monitoring/runbooks/nuxt_avg_response_time_above_threshold.md:7]:
```suggestion
- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/Nuxt+Production+AVG+response+time+above+threshold?>
```

File: documentation/meta/monitoring/runbooks/nuxt_p99_response_time_above_threshold.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Run Book: Nuxt Production Average Response Time above threshold
```
Comment (obulat) [inline documentation/meta/monitoring/runbooks/nuxt_p99_response_time_above_threshold.md:1]:
```suggestion
# Run Book: Nuxt Production P99 Response Time above threshold
```

**Pull request merged**